aemedge/blocks/table/table.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Table Block
```

Comment:
Comments

Reply:
Fixed, took that over from boilerplate project.